activiti-core/activiti-image-generator/src/main/java/org/activiti/image/impl/DefaultProcessDiagramGenerator.java

```diff
@@ -1131,6 +1153,27 @@ private static void drawGatewayHighLightCompleted(DefaultProcessDiagramCanvas pr
         processDiagramCanvas.drawGatewayHighLightCompleted(graphicInfo);
     }
 
+    private static void drawTaskHighLightErrored(DefaultProcessDiagramCanvas processDiagramCanvas,
+        GraphicInfo graphicInfo) {
+        processDiagramCanvas.drawHighLightErrored((int) graphicInfo.getX(),
+            (int) graphicInfo.getY(),
+            (int) graphicInfo.getWidth(),
+            (int) graphicInfo.getHeight());
```

Comment:
I'd prefer to have the entire `GraphicInfo` passed as a parameter, instead of splitting it into several parameters.